src/main/java/cz/datadriven/utils/config/view/ConfigViewFactory.java

```diff
@@ -42,22 +55,32 @@
    * @param <T> type of the view class to be created
    * @return the view
    */
-  @SuppressWarnings("unchecked")
   public static <T> T create(Class<T> configViewClass, Config config) {
     if (!ConfigViewProxy.canProxy(configViewClass)) {
       throw new IllegalArgumentException(
-          "Can not instantiate ConfigView for "
-              + "class [ "
-              + configViewClass
-              + " ]. Did you forget @ConfigView annotation?");
+          String.format(
+              "Can not instantiate ConfigView for class [%s]. Did you forget @ConfigView annotation?",
+              configViewClass));
     }
-    final Enhancer enhancer = new Enhancer();
-    enhancer.setCallback(new ConfigViewProxy(new ConfigViewProxy.Factory(config)));
-    if (configViewClass.isInterface()) {
-      enhancer.setInterfaces(new Class[] {configViewClass});
-    } else {
-      enhancer.setSuperclass(configViewClass);
+    try {
+      return new ByteBuddy(ClassFileVersion.JAVA_V8)
```

Comment:
How does this relate to `sourceCompatibility` and `targetCompatibility` in build.gradle? Some comment could be added to show if the versions are independent, have to be the same, or something in-between.

Reply:
This tells byte-buddy to generate Java 8 compatible byte code. This is basically minimal java version we want to support, I'll add some comment with an explanation.